config/karma.conf.js

```diff
@@ -87,7 +87,7 @@ module.exports = function(config) {
      * Continuous Integration mode
      * if true, Karma captures browsers, runs the tests and exits
      */
-    singleRun: true
+    singleRun: false
```

Comment:
I wouldn't change this. In my opinion, `singleRun: true` is better if using vagrant or any similar VM approach because it prints the output to stdout. People wanting CI can always change it.